src/core/drive/form_submission.ts

```diff
@@ -24,6 +24,20 @@ export enum FormSubmissionState {
   stopped,
 }
 
+enum FormEnctype {
+  urlEncoded = "application/x-www-form-urlencoded",
+  multipart  = "multipart/form-data",
+  plain      = "text/plain"
+}
+
+function formEnctypeFromString(encoding: string): FormEnctype {
+  switch(encoding.toLowerCase()) {
```

Comment:
@kaspth I think reverse mapping would be great for our FetchMethod enum, but it seems to break down when there are `/` in the String value.

Reply:
Oh huh! My understanding from the docs was that it was basically like `FormEnctype[FormEnctype[urlEncoded] = "application/x-www-form-urlencoded"] = urlEncoded` and I don't get why that wouldn't work. Oh well, maybe something at the transpilation stage 😄

Reply:
Ah! There's this bit in the documentation 😢

```
Keep in mind that string enum members do not get a reverse mapping generated at all.
```